challenges/models.py

```diff
@@ -28,7 +32,7 @@ class CategoryChallenge(models.Model):
     challenge = models.ForeignKey(Challenge, on_delete=models.CASCADE)
 
     def __str__(self):
-        return self.category.name + ': ' + self.challenge.name
+        return self.category.name + ": " + self.challenge.name
```

Comment:
```suggestion
        return f"{self.category}: {self.challenge}"
```

W obu obiektach zdefiniowałaś `__str__` :)

Reply:
😍